src/GafferSceneUI/TranslateTool.cpp

```diff
@@ -116,6 +116,12 @@ void TranslateTool::updateHandles()
 {
 	handles()->setTransform( handlesTransform() );
 
+	// Because we provide multiple orientations, the handles
+	// may well not be aligned with the axes of the transform
+	// space. So any given handle might affect several components
+	// of the target translation. For each handle, check to see
+	// if each of the plugs it effects are settable, and if not,
+	// disable the handle.
```

Comment:
Maybe having the orientation default to World would have avoided the confusion on my end. Basically, the workflow was:

- go to node graph
  - drop down a cube
- go to node editor
  - rotate in y
  - gang translate
- go to viewport
  - turn on TranslateTool
    - gah! i cant do anything
- go back to node editor
  - actually, i can do exactly what i want by scrubbing translate.x
- go back to viewport
    - flip orientation to world
    - now i can do what i want
      - and it matches what was happening when i was scrubbing in the node editor